sfxclient/sfxclient.go

```diff
@@ -175,8 +206,23 @@ func (s *Scheduler) Var() expvar.Var {
 func (s *Scheduler) CollectDatapoints() []*datapoint.Datapoint {
 	ret := make([]*datapoint.Datapoint, 0, len(s.previousDatapoints))
 	now := s.Timer.Now()
-	for _, p := range s.callbackMap {
-		ret = append(ret, p.getDatapoints(now, s.SendZeroTime)...)
+	if s.debug {
+		parentSpan := opentracing.GlobalTracer().StartSpan("collect-datapoints")
+		var span opentracing.Span
+		for group, p := range s.callbackMap {
+			if group == "" {
```

Comment:
small nit, i'd probably just override group with "default-callback-group" so you're not duplicating the span creation line...

Reply:
thanks for the suggestion. Done